client/src/configs/token.js

```diff
@@ -0,0 +1,3 @@
+const token =
+  'eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJodHRwOlwvXC9wZmEuZm9yZWNhLmNvbVwvYXBpXC92MVwvXC9hdXRob3JpemVcL3Rva2VuIiwiaWF0IjoxNjUxMjE0NTgxLCJleHAiOjk5OTk5OTk5OTksIm5iZiI6MTY1MTIxNDU4MSwianRpIjoiMTI0ZWNlOTg5MGY5ZDRiOCIsInN1YiI6Imx1a29uZWxpczEiLCJmbXQiOiJYRGNPaGpDNDArQUxqbFlUdGpiT2lBPT0ifQ.yrTZFMKWGHro_ZPBIBRgQ1E8i36quPRZb4NJDCkRkx8';
```

Comment:
still see hardcoded token, please remove it

Reply:
hard coded token removed

Reply:
Do you still need this file?